monitor_mqtt/monitor_mqtt.yaml.example

```diff
@@ -0,0 +1,22 @@
+### Place configuration data for your pack here.
+---
+hostname: ''
```

Comment:
The example file should have example data in it. Would be best to put "real" things in here. Obviously you can use fake usernames/passwords, but it would be nice to have actual examples of the data in the other fields.

Reply:
Added example data